lib/help_scout.rb

```diff
@@ -137,9 +137,30 @@ def create_thread(conversation_id:, thread:, imported: nil, reload: nil)
 
     post("conversations/#{conversation_id}", body: thread, query: query)
 
+    return last_response if reload
```

Comment:
Should this indeed be the full response, and not something like `last_response.body`? 

I've asked @djfpaagman to take a look at this as well because he's more familiar with the reload stuff in the places where we use this gem. I'm not super sure if reloading is required/used already, and how.

Reply:
Hmm i'm a bit torn. On the one hand I want to let the HS API determine what is returned as much as possible, and if you `reload: true` it automatically returns the data. But we already have this explicit `last_response.code == HTTP_CREATED` as it is. I guess I like something like this better:

```ruby
if reload
  last_response.parsed_response
else
  last_response.code == HTTP_OK
end
```
To make the flow explicit and easy to spot that there are two possible return values.